eng/Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+
+    <ItemGroup>
+        <ProjectToBuild Include="$(RepoRoot)Build.proj" />
+    </ItemGroup>

```

Comment:
style nit:

```suggestion
  <ItemGroup>
    <ProjectToBuild Include="$(RepoRoot)Build.proj" />
  </ItemGroup>

```

Reply:
Thanks. My VS Code keeps reverting to adding 4 spaces instead of 2 in xmls, targets, props and csprojs. It drives me crazy.

Reply:
Yeah because this repo is missing a .gitattributes file